### PR TITLE
feat: boostrap of neo4j and simple query

### PR DESCRIPTION
**Pull request by thomas-mauran**
boostrap of neo4j

### CODE REVIEW

File: backend/api_gateway/src/main/kotlin/com/linkedout/backend/controller/RecommendationController.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Flux
+
+@RestController
+@RequestMapping("/api/v1/recommendations")
+open class RecommendationController(private val recommendationService: RecommendationService) {
+    @GetMapping
+    open fun getJobs(request: ServerHttpRequest): Flux<Recommendation> {
+        println("HERE")
```
Comment (Kuruyia) [inline backend/api_gateway/src/main/kotlin/com/linkedout/backend/controller/RecommendationController.kt:16]:
```suggestion
```

File: backend/api_gateway/src/main/kotlin/com/linkedout/backend/service/RecommendationService.kt [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        val getRecommendationResponse = response.getRecommendationResponse
+        return getRecommendationResponse.recommendationsList.map { recommendation ->
+            Recommendation(
+                recommendation.id
+            )
+        }
+    }
+}
+
+private fun convertJobOfferFromProto(source: JobOfferOuterClass.JobOffer): JobOffer {
```
Comment (Kuruyia) [inline backend/api_gateway/src/main/kotlin/com/linkedout/backend/service/RecommendationService.kt:44]: Where is this used?

File: backend/protobuf/src/main/proto/request.proto [hunk 3]
```diff
@@ -67,5 +68,6 @@ message Request {
     services.SetUserProfilePictureRequest set_user_profile_picture_request = 54;
     services.GetUserCvRequest get_user_cv_request = 55;
     services.SetUserCvRequest set_user_cv_request = 56;
+    services.GetRecommendationRequest get_user_recommendation = 57;
```
Comment (Kuruyia) [inline backend/protobuf/src/main/proto/request.proto:71]:
```suggestion
    services.GetRecommendationRequest get_recommendation_request = 57;
```

**Review by Kuruyia — CHANGES_REQUESTED** [review]

File: backend/api_gateway/src/main/kotlin/com/linkedout/backend/service/RecommendationService.kt (hunk 2 above)
Comment (thomas-mauran) [reply backend/api_gateway/src/main/kotlin/com/linkedout/backend/service/RecommendationService.kt:44]: unused removed

**Review by Kuruyia — APPROVED** [review]

**Pull request merged**